roles/matrix-bridge-mautrix-signal/templates/systemd/matrix-mautrix-signal-db.service.j2

```diff
@@ -0,0 +1,32 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Mautrix Signal database server
+
+[Service]
+Type=simple
+
+ExecStartPre=-{{ matrix_host_command_docker }} kill {{ matrix_mautrix_signal_db_host }}
+ExecStartPre=-{{ matrix_host_command_docker }} rm {{ matrix_mautrix_signal_db_host }}
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre={{ matrix_host_command_sleep }} 5
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name {{ matrix_mautrix_signal_db_host }} \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--network={{ matrix_docker_network }} \
+			--env POSTGRES_USER={{ matrix_mautrix_signal_db_user }} \
+			--env POSTGRES_PASSWORD={{ matrix_mautrix_signal_db_password }} \
+			--env POSTGRES_DB={{ matrix_mautrix_signal_db_database }} \
+			-v {{ matrix_mautrix_signal_db_storage_path }}:/var/lib/postgresql/data:z \
```

Comment:
I had to add "-v /etc/passwd:/etc/passwd:ro" on the server for this to work. Otherwise it failed in chmod /var/run/postgres.

Reply:
Thanks for spotting that! I must have missed this during testing. I found [this piece of documentation](https://github.com/docker-library/docs/tree/2c7f3a40774774389a4a372ef21ccd34ea53d551/postgres#arbitrary---user-notes) that you probably also found, which mentions the work-around you suggested as well as a second option. I committed your proposal as it was faster to do and shouldn't have any negative side-effects, even though the other options seems a bit cleaner.